lib/publiccloud/ec2.pm

```diff
@@ -208,8 +208,9 @@ sub teardown {
 
 sub describe_instance {
     my ($self, $instance_id, $query) = @_;
+    my $region = get_required_var('PUBLIC_CLOUD_REGION');
```

Comment:
Do we really need to have this variable as required? Isn't there a default value that we use otherwise, when this is not defined? :thinking: